pyproject.toml

```diff
@@ -55,6 +55,7 @@ dev = [
     "pandas-stubs",
     "pyarrow-stubs",
     "pydocstyle",
+    "PyQt5-sip<12.16.0",
```

Comment:
What was the reasoning for this?

Reply:
Yes, sorry, a bit arbitrary without comment. This is a dependency in Ert that just dropped Python 3.8 an hour ago, so Python 3.8 CI will fail if not pinned to one that still supports it. Some extra context is in Slack